xbmc/GUIInfoManager.cpp

```diff
@@ -9133,13 +9133,13 @@ int CGUIInfoManager::TranslateSingleString(const std::string &strCondition, bool
       {
         int position = atoi(info[1].param().c_str());
         int value = TranslateMusicPlayerString(info[2].name); // musicplayer.position(foo).bar
-        return AddMultiInfo(CGUIInfo(value, 0, position));
+        return AddMultiInfo(CGUIInfo(value, 2, position)); // 2 => absolute (0 used for not set)
       }
       else if (info[1].name == "offset")
       {
         int position = atoi(info[1].param().c_str());
         int value = TranslateMusicPlayerString(info[2].name); // musicplayer.offset(foo).bar
-        return AddMultiInfo(CGUIInfo(value, 1, position));
+        return AddMultiInfo(CGUIInfo(value, 1, position));  // 1 => relative
```

Comment:
> Do not use whitespace to vertically align [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#35-vertical-alignment

Reply:
No attempt at alignment here, so what is the issue?

Reply:
two spaces before `//` so that `//` is aligned with the above line

Reply:
So you are saying one and only one space before comments? 
OK, but personally this "pretty code" thing is becoming a little extreme.